src/fides/api/api/v1/endpoints/health.py

```diff
@@ -54,6 +57,20 @@ def get_cache_health() -> str:
         return "unhealthy"
 
 
+def is_email_messaging_enabled(db: Session) -> bool:
```

Comment:
per my comment below, may we want this function to actually look at the messaging service that's actively configured to be _used_, i.e. by looking at the `notifications.notification_service_type` config property? and we should be looking at the config proxy (i know that's a bit of a "gotcha"!). 

we've actually built an endpoint that does something very similar to this - check out `get_messaging_status` in `messaging_endpoints.py` - that's evaluating whether a messaging service is fully configured and "ready to go". i think we want a similar check here, but we want to also ensure that this actively configured messaging service is an email messaging service.

if we don't actually verify that the email service is fully configured to be used, then i think there's an edge case here where we set users up for failur e- unless i'm missing something!